DEVELOPMENT.md

```diff
@@ -76,7 +76,12 @@ Please refer to [Contributing via Pull Requests](./CONTRIBUTING.md#contributing-
 4. Start using `rush cinstall` or `rush cupdate` instead of `rush install` or `rush update`.
 
 ### Add packages using rush:
-1. `rush add -p <packageName> --caret -s; rush cupdate`
+1. Add package to 1 project: `cd <TO_PROJECT_DIR>; rush add -p <packageName> --caret -s; rush cupdate`
+2. Add to all projects: `rush add -p <packageName> --caret -s --all --make-consistent; rush cupdate`
```

Comment:
should this be a common occurrence?

Reply:
Its anecdotal info, but I've run into it a couple times 🤷.

Reply:
@bhudnell removed !